trusted-firmware-m/platform/ext/target/nordic_nrf/common/core/gcc/nordic_nrf_s.ld

```diff
@@ -67,6 +77,14 @@ MEMORY
             <= CMSE_VENEER_REGION_SIZE, "Veneer region overflowed")
 #endif
 
+#if !defined(BL2) || (MCUBOOT_IMAGE_NUMBER == 1)
+#define VENEER_POS 3
```

Comment:
@hakonfam can you add some inline explanations for the different VENEER placements?
perhaps an one-line comment for each of the options 1, 2, 3